sentence.go

```diff
@@ -86,12 +92,14 @@ func parseSentence(raw string) (BaseSentence, error) {
 			"nmea: sentence checksum mismatch [%s != %s]", checksum, checksumRaw)
 	}
 	talker, typ := parsePrefix(fields[0])
+
```

Comment:
Remove blank lines. If separate blocks of code need to visually separated, add a comment between them.

Reply:
@icholy done: 70373c0